SimpleAkkaDotNetCluster/SimpleClusterCaller/ClusterClientActor.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Immutable;
+using Akka.Actor;
+using Akka.Cluster.Tools.Client;
+using Akka.Event;
+using SimpleClusterLib;
+
+namespace SimpleClusterCaller;
+
+public class ClusterClientActor: ReceiveActor
+{
+    public static Props Props(ImmutableHashSet<ActorPath> contactPoints)
+        => Akka.Actor.Props.Create(() => new ClusterClientActor(contactPoints));
+    
+    private readonly ImmutableHashSet<ActorPath> _contactPoints;
+    private readonly ILoggingAdapter _log;
+    private IActorRef _clusterClient;
+
+    public ClusterClientActor(ImmutableHashSet<ActorPath> contactPoints)
+    {
+        _contactPoints = contactPoints;
+        _log = Context.GetLogger();
+
+        Receive<ServiceMessage>(message =>
+        {
+            _clusterClient.Tell(new ClusterClient.Send(
+                "/user/" + message.ServiceName, new PayloadEnvelope(Sender, message.Payload)));
```

Comment:
For `ClusterClient` to address an actor inside a cluster, it __must__ be in the form of "/user/__actorName__" where __actorName__ is the name of the actor that has been registered to a `ClusterClientReceptionist` inside the cluster.

Reply:
Note that here we're creating an envelope class "PayloadEnvelope", its purpose is to add a reference to the original sender as a metadata to the payload.